coremltools/converters/mil/frontend/torch/test/test_torch_ops.py

```diff
@@ -2366,6 +2368,26 @@ def forward(self, x):
 
         self.run_compare_torch(shape, OnesStaticModel().eval(), backend=backend)
 
+class TestRandint(TorchBaseTest):
+    @pytest.mark.parametrize(
+        "backend, shape, low, high",
+        itertools.product(
+            backends,
+            [(1,), (2, 3)],
+            [-1, 2],
+            [3, 5],
+        ),
+    )
+    def test_randint(self, backend, shape, low, high):
+        class Randint(nn.Module):
+            def forward(self, x):
+                return torch.randint(low, high, x.shape)
+
+        torch_in = torch.tensor(shape, dtype=torch.int32)
+        model = Randint().eval()
+        verify = lambda x: numpy.testing.assert_array_less(x, high) and numpy.testing.assert_array_less(low, x)
```

Comment:
I do like checking that `low < x < high`. However I don't think it justifies adding a lot more testing functionality.

If you want, you can do the model conversion yourself in this function. Then get the output and do this check.

It's also fine if you just want to modify the Torch model to return the length. Then use our existing testing helper functions.